src/main/java/frc/robot/RobotContainer.java

```diff
@@ -107,6 +108,20 @@ private void configureButtonBindings() {
    * @return the command to run in autonomous
    */
   public Command getAutonomousCommand() {
-    return null;
+    /*
+     * The plan for two ball autonomous mode:
+     * Starting Position: facing towards the ball, at the corner parallel to the
+     * line
+     * 1. Go to ball (drive forwards 1 meter)
+     * 2. Run intake for 1 second
+     * 2. Turn around and get into position for shooting the ball (drive field
+     * relative backwards 3 meters, turn 180-ish degrees)
+     * 3. Shoot the ball
+     */
+
+    // TODO ADD INTAKE COMMAND AND SHOOT COMMAND TO THE SEQUENCE
+    return new SequentialCommandGroup(
+        new MoveCommand(m_swerveDriveSubsystem).withRobotRelativeX(1),
+        new LimelightAimingCommand(new MoveCommand(m_swerveDriveSubsystem).withFieldRelativeX(-3), 0));
```

Comment:
one concern i have is if the limelight loses sight of the target before it finishes rotating, just because of the angle